create-site-map.js

```diff
@@ -0,0 +1,66 @@
+const unirest = require("unirest");
+
+const pluginSite = process.env.REST_API_URL || "https://plugins.jenkins.io/api";
+
+function formatDate(date) {
+  var d = new Date(date),
+    month = "" + (d.getMonth() + 1),
+    day = "" + d.getDate(),
+    year = d.getFullYear();
+
+  if (month.length < 2) month = "0" + month;
+  if (day.length < 2) day = "0" + day;
+
+  return [year, month, day].join("-");
+}
+
+function getPage(page) {
+  const url = `${pluginSite}/plugins/?limit=100&page=${page}`;
+  console.info(`Fetching '${url}'`);
+  return unirest.get(url).then(results => {
+    if (results.statusCode !== 200) {
+      throw results.body;
+    }
+    return results.body;
+  });
+}
+
+function createSiteMap() {
+  const plugins = [];
+
```

Comment:
is it worth adding a log message, something like:
```suggestion
console.info('Starting to fetch plugins for sitemap')
```

I just ran it in my terminal and it would be unclear why i'm getting all these log messages:
```
Fetching 'https://plugins.jenkins.io/api/plugins/?limit=100&page=2'
Fetching 'https://plugins.jenkins.io/api/plugins/?limit=100&page=3'
(node:8281) [DEP0066] DeprecationWarning: OutgoingMessage.prototype._headers is deprecated
Fetching 'https://plugins.jenkins.io/api/plugins/?limit=100&page=4'
Fetching 'https://plugins.jenkins.io/api/plugins/?limit=100&page=5'
Fetching 'https://plugins.jenkins.io/api/plugins/?limit=100&page=6'
Fetching 'https://plugins.jenkins.io/api/plugins/?limit=100&page=7'
Fetching 'https://plugins.jenkins.io/api/plugins/?limit=100&page=8'
Fetching 'https://plugins.jenkins.io/api/plugins/?limit=100&page=9'
Fetching 'https://plugins.jenkins.io/api/plugins/?limit=100&page=10'
Fetching 'https://plugins.jenkins.io/api/plugins/?limit=100&page=11'
Fetching 'https://plugins.jenkins.io/api/plugins/?limit=100&page=12'
Fetching 'https://plugins.jenkins.io/api/plugins/?limit=100&page=13'
Fetching 'https://plugins.jenkins.io/api/plugins/?limit=100&page=14'
Fetching 'https://plugins.jenkins.io/api/plugins/?limit=100&page=15'
Fetching 'https://plugins.jenkins.io/api/plugins/?limit=100&page=16'
Fetching 'https://plugins.jenkins.io/api/plugins/?limit=100&page=17'
Fetching 'https://plugins.jenkins.io/api/plugins/?limit=100&page=18'
```